### PR TITLE
React/week1

### PR DESCRIPTION
**Pull request by Kiran-Serolla**
implemented week1 homework

### CODE REVIEW

File: react/week1/todo-list/src/App.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```
Comment (amjadiftikhar) [inline react/week1/todo-list/src/App.js:1]: @Kiran-Serolla, over all nice job done

File: react/week1/todo-list/src/App.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
+.App {
```
Comment (amjadiftikhar) [inline react/week1/todo-list/src/App.css:1]: Over all nice job done. 

Your can remove default styles for the App!

File: react/week1/todo-list/src/App.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import logo from './logo.svg';
```
Comment (amjadiftikhar) [inline react/week1/todo-list/src/App.js:2]: add some styles to your app!

File: react/week1/todo-list/src/todo.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (amjadiftikhar) [inline react/week1/todo-list/src/todo.js:1]: No need to import react-dom here!

File: react/week1/todo-list/src/todo.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+import ReactDOM from 'react-dom';
+
+
+const ToDo = (props) => {
+
```
Comment (amjadiftikhar) [inline react/week1/todo-list/src/todo.js:6]: Better to add some styles to this todo list.

**Review by amjadiftikhar — COMMENTED** [review]
Great job, some small changes need to be made!

File: react/week1/todo-list/src/todo.js (hunk 5 above)
Comment (Kiran-Serolla) [reply react/week1/todo-list/src/todo.js:5]: Implemented styled

File: react/week1/todo-list/src/App.css (hunk 2 above)
Comment (Kiran-Serolla) [reply react/week1/todo-list/src/App.css:1]: done

**Pull request merged**